lib/src/ovis_json/ovis_json_parser.y

```diff
@@ -109,9 +109,11 @@ attr_list: /* empty */ { $$ = new_dict_val(); }
 	 | string ':' value {
 	     json_entity_t e = new_dict_val();
 	     $$ = add_dict_attr(e, $1, $3);
+	     json_entity_free($1);
 	 }
 	 | attr_list ',' string ':' value {
 	     $$ = add_dict_attr($1, $3, $5);
+	     json_entity_free($3);
```

Comment:
@tom95858 With the same reason above, this is also legit.

Reply:
@nichamon is there an input file that exercises the entire grammar (lex and yacc) that we can test routinely as the config tools evolve?